docs/index.asciidoc

```diff
@@ -68,6 +68,12 @@ Further documentation describing this syntax can be found
 https://github.com/jmettraux/rufus-scheduler#parsing-cronlines-and-time-strings[here].
 
 
+[id="plugins-{type}s-{plugin}-auth"]
+==== Authentication
+
+Authentication to a secure Elasticsearch cluster is possible using one of the `user`/`password`, `cloud_auth` or `api_key` options.
```

Comment:
Same suggestion as in logstash-plugins/logstash-filter-elasticsearch#132

```suggestion
Authentication to a secure Elasticsearch cluster is possible using _one_ of the following options:
* <<plugins-{type}s-{plugin}-user>> AND <<plugins-{type}s-{plugin}-password>>
* <<plugins-{type}s-{plugin}-cloud_auth>>
* <<plugins-{type}s-{plugin}-api_key>>
```

Reply:
I'll defer to @karenzone on this!

Reply:
Suggested content is good and will be helpful to users.  Formatting isn't quite right.  For new content to render correctly, you need a return after the colon on line 71.

Reply:
I've updated the suggestion with the newline before the list.

Reply:
Fantastic. Thanks!